packages/web-fragments/src/gateway/middlewares/netlify/index.ts

```diff
@@ -0,0 +1,195 @@
+import { HTMLRewriter } from "https://ghuc.cc/worker-tools/html-rewriter/index.ts";
```

Comment:
this file is the one that matters the most. it has the right shape but I can't review thoroughly now, so I'll do more pass on it tomorrow.